### DISCUSSION TITLE
Magiczoom instead of fotoroma

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#123] [3. Integrations]
Hello 
I would like to integrate magic zoom which is 5x faster more than Magento core gallery, and it has more feature  
I have attached here magiczoom could you check that proper for breeze or not ?

Here is the extension link
https://drive.google.com/drive/folders/10VrwJyjNXN6qg2hfK-Wrw6Edc0QBYQCQ?usp=sharing

about magiczoom 
https://github.com/magento/magento2/issues/6018#issuecomment-250767500

**Comment by vovayatsyuk** [comment]
We do not use fotorama, we use our own gallery script. For now, we are not interested in magiczoom integration.

What features are you missing from our gallery?
